includes/admin/components/bouncer/wsl.components.bouncer.setup.php

```diff
@@ -70,7 +70,6 @@ function wsl_component_bouncer_setup_wsl_widget()
 				<select name="wsl_settings_bouncer_registration_enabled">
 					<option <?php if( get_option( 'wsl_settings_bouncer_registration_enabled' ) == 1 ) echo "selected"; ?> value="1"><?php _wsl_e("Yes", 'wordpress-social-login') ?></option>
 					<option <?php if( get_option( 'wsl_settings_bouncer_registration_enabled' ) == 2 ) echo "selected"; ?> value="2"><?php _wsl_e("No", 'wordpress-social-login') ?></option> 
-					<option <?php if( get_option( 'wsl_settings_bouncer_registration_enabled' ) == 3 ) echo "selected"; ?> value="3"><?php _wsl_e("No, but allow existing users to connect", 'wordpress-social-login') ?></option>
```

Comment:
@miled I removed this option as currently, is not being used at all in code and the next field is about `Allow authentication`. I think a mix of the values of these 2 fields can give the same result. 

Let me know if you agree, otherwise, I can revert this change.